src/org/janelia/mipav/plugins/worm/untwisting/PlugInDialogVolumeRenderDualJanelia.java

```diff
@@ -2164,6 +2207,129 @@ private void annotationAnimationFromSpreadSheet() {
 		progress = null;
 	}
 	
+	private JFreeChart selectionChart;
+	private ChartPanel chartPanel;
+	private List<Vector3f> chart3DPoints;
```

Comment:
We may want to consider one of two things:
1) Moving this to the top
2) Factoring this out into a separate class.

Reply:
1. resolved
2. how to factoring this into a separate class?

Reply:
- [ ] created a new class SelectionChartPanel to factoring above